### PR TITLE
docs: update package root readme files

### PR DESCRIPTION
**Pull request by viterobk**

### CODE REVIEW

File: packages/dx-react-grid/README.md [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
+- [Getting Started](#getting-started)
+- [Plugin Overview](plugin-overview)
+- [Controlled and Uncontrolled modes](controlled-and-uncontrolled-modes)
+- [Guides by Features](guides)
+- [Plugin Reference](reference)
+
 ## Overview
 
-The DevExtreme React Grid component has a composable and extendable plugin-based architecture that allows you to display table data with a set of different transformations, such as paging, sorting, filtering, grouping, etc. It also allows row selection and data editing, supports the controlled and uncontrolled state modes, and can be used in either a regular or a Redux-based application. It comes with Twitter Bootstrap rendering and theming out of the box.
+DevExtreme React Grid is a component that displays table data from a local or remote source. It supports paging, sorting, filtering, grouping and other data shaping options, row selection, and data editing. Support for controlled and uncontrolled state modes allows you to use the Grid in a regular or Redux-based application. The DevExtreme Grid component has a composable and extendable plugin-based architecture and is provided with Twitter Bootstrap rendering and theming out of the box.
```
Comment (dxbykov) [inline packages/dx-react-grid/README.md]: Now it's provided with Material UI rendering and theming as well.

File: packages/dx-react-grid/README.md [hunk 2]
```diff
@@ -1,41 +1,57 @@
 # DevExtreme React Grid
 
+- [Overview](#overview)
```
Comment (dxbykov) [inline packages/dx-react-grid/README.md]: I think it's better not have local and cross-page links in the same menu.

File: packages/dx-react-grid/README.md [hunk 3]
```diff
@@ -59,10 +75,17 @@ const App = () => (
...
+### Try Out The React Grid
+
+Following the links below to try out the React Grid:
+- [WebpackBin for Bootstrap3](https://www.webpackbin.com/bins/-KoCcFwPPtps5fEN0PkI)
+- [WebpackBin for Material UI](https://www.webpackbin.com/bins/-KoCeS-_jZrZuTecv-wd)
+
 ## Documentation
 
-[React Grid Documentation](https://devexpress.github.io/devextreme-reactive/react/grid/docs/)
+[Guides](https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides)
```
Comment (dxbykov) [inline packages/dx-react-grid/README.md]: We should use a trailing hash with the cross-page links.

File: packages/dx-react-grid/README.md [hunk 4]
```diff
@@ -1,41 +1,57 @@
 # DevExtreme React Grid
 
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+- [Plugin Overview](plugin-overview)
```
Comment (dxbykov) [inline packages/dx-react-grid/README.md]: These links will be broken once the page is published on npm. We should use absolute URLs for the root readme files.

File: packages/dx-react-grid/README.md (hunk 2 above)
Comment (viterobk) [inline packages/dx-react-grid/README.md]: We have removed this menu.

File: packages/dx-react-grid/docs/README.md [hunk 5]
```diff
@@ -75,136 +64,17 @@ const App = () => (
...
-
-const App = () => (
-  <Grid rows={[/* ... */]} columns={[/* ... */]}>
-    <FilteringState defaultFilters={[/* ... */]}/>
-    <LocalFiltering/>
-    <TableView/>
-  </Grid>
-);
-```
+## Documentation
```
Comment (kvet) [inline packages/dx-react-grid/docs/README.md]: Is this section needed on the site?

Comment (viterobk) [inline packages/dx-react-grid/docs/README.md]: Looks like redundant.

**Pull request merged**